blog/.vuepress/components/Categories.vue

```diff
@@ -5,11 +5,11 @@
       <li v-for="category in $category.list">
 
           <span v-if="$page.frontmatter.title == (category.name + ' ')" class="active-link">
-            {{ category.name }}
+            {{ category.name.replace(/^\w/, (c) => c.toUpperCase()) }}
```

Comment:
fyi: Vue has the concept of [filters](https://vuejs.org/v2/guide/filters.html) exactly for this.

Reply:
Thanks Dennis -- I addressed this in the latest commits

Reply:
Great, looks good! 💪